gradle.properties

```diff
@@ -18,4 +18,8 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+#Access Key
+CLIENT_ID = "RUvTzcL-6uXdrLTWpudHjLa9MqIvBcjPmJ8bqJwFpQg"
+
+BASE_URL = "https://api.unsplash.com/"
```

Comment:
it can be removed from here, because if we use this approach we can't use a separate endpoint and api key for the paid version